src/time.c

```diff
@@ -46,6 +46,61 @@ rcutils_time_point_value_as_nanoseconds_string(
   return RCUTILS_RET_OK;
 }
 
+rcutils_ret_t
+rcutils_time_point_value_as_date_string(
+  const rcutils_time_point_value_t * time_point,
+  char * str,
+  size_t str_size)
+{
+  RCUTILS_CHECK_ARGUMENT_FOR_NULL(time_point, RCUTILS_RET_INVALID_ARGUMENT);
+  RCUTILS_CHECK_ARGUMENT_FOR_NULL(str, RCUTILS_RET_INVALID_ARGUMENT);
+  if (0 == str_size) {
+    return RCUTILS_RET_OK;
+  }
```

Comment:
Out of curiosity, why is it RCUTILS_RET_OK if the str_size is 0?  I would think we could move the check to ensure that `str_size` >= 32 to here, but maybe there is a reason I'm not remember that an empty str_size is OK.

Reply:
I don't know, I just copied this from the other functions